src/devices/esp8266.c

```diff
@@ -109,11 +109,21 @@ static void ipd_urc_cb(char *msg)
 	 * comes from the command messages.
 	 */
 	if (serial_len < len) {
-		pr_error(LOG_PFX "IPD Length Mismatch.  Dropping. \r\n");
-		pr_info_int_msg(LOG_PFX "Length Expected: ", (int) len);
-		pr_info_int_msg(LOG_PFX "Length Actual: ", (int) serial_len);
-		pr_info_str_msg(LOG_PFX "Data: ", data);
-		return;
+
+		/* There's more data coming */
```

Comment:
Rationale: The camera's webserver sends a multi-line response. If there's less data than expected on the initial response, we need to siphon the rest of the data until complete.